cli/js/files.ts

```diff
@@ -231,5 +281,82 @@ export type OpenMode =
  * @internal
  */
 export function create(filename: string): Promise<File> {
-  return open(filename, "w+");
+  return open(filename, {
+    read: true,
+    create: true,
+    write: true,
+    truncate: true
+  });
+}
+
+/** Check if OpenMode is set to something that is valid.
+ *  @returns Tuple representing if openMode is valid and error message if it's not
+ *  @internal
+ */
+function checkOpenMode(mode: OpenMode): [boolean, string] {
```

Comment:
If you skip this validation and just forward "invalid" values to Rust would it error out?

Reply:
It would error
``` bash
> error: Uncaught InvalidInput: Invalid argument (os error 22)
► $deno$/dispatch_json.ts:40:11
    at DenoError ($deno$/errors.ts:20:5)
    at unwrapResponse ($deno$/dispatch_json.ts:40:11)
    at sendAsync ($deno$/dispatch_json.ts:85:10)
```